src/helpers/dataLoading/plans.ts

```diff
@@ -0,0 +1,25 @@
+import { toast } from 'react-toastify';
+import { ActionCreator, AnyAction } from 'redux';
+import { OPENSRP_PLANS_BY_USER_FILTER } from '../../constants';
+import { OpenSRPService } from '../../services/opensrp';
+import store from '../../store';
+import { growl } from '../utils';
+
+/** find plans that the given OpenMRS userName has access to
+ * @param {string} userName - openMRSUserName
```

Comment:
We should not mention `openMRS` at all anywhere in my view.

Reply:
sawa